cabal.project

```diff
@@ -0,0 +1,12 @@
+optional-packages: miso
```

Comment:
I would have expected to be able to use `optional-packages: . miso` here, but for some reason the frontend build then still complains about `ihp` being an unknown library.